components/item/AwesomeSwiper.vue

```diff
@@ -74,30 +76,43 @@
       swiper() {
         return this.$refs.awesome.$swiper;
       }
-    },
-    mounted() {
-      // console.log('Current Swiper instance object', this.swiper)
-      // this.swiper.slideTo(3, 1000, false)
     }
   };
 </script>
 
 <style lang="scss">
+  @import './assets/scss/variables.scss';
+
   .swiper-container {
     max-height: 568px;
     height: 55vh;
   }
   .swiper-slide {
     width: auto;
+    :before {
+      content: '';
+      transition: $standard-transition;
+    }
+    &:not(.swiper-slide-active):before {
+      content: '';
+      width: 100%;
+      left: 0;
+      top: 0;
+      height: 100%;
+      position: absolute;
+    }
     a {
       display: inline-flex;
       height: 100%;
       align-items: center;
     }
+    &:last-of-type {
+      margin-right: 50vw !important;
+    }
```

Comment:
As discussed, could you remove this for now?

Reply:
ok